app/client/src/git/sagas/commitSaga.ts

```
@@ -42,23 +45,23 @@ export default function* commitSaga(
-    if (
-      GitErrorCodes.REPO_LIMIT_REACHED === response?.responseMeta?.error?.code
-    ) {
-      yield put(
-        gitArtifactActions.toggleRepoLimitErrorModal({
-          ...basePayload,
-          open: true,
-        }),
-      );
-    }
+  } catch (e) {
+    if (response && response.responseMeta.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)